### DISCUSSION TITLE
UI / Easier install plans?

### DISCUSSION THREAD

**Discussion by Damienlee69** [jmbannon/ytdl-sub#578] [General]
Just wondering if there's any plans for a UI? I've dabbled a bit in CLI when needed and sometimes get stuff running. Anyways, I know it says, be comfortable with CLI 'n' such, but a UI is just easier for the majority I think. Honestly I think i was (kind of) close to getting going, though I had to change dir structure cuz I'm dumb I suppose. I ended up having everything in c:\ytdl-sub with only a sub folder for the working dir. Made my own yamls, copied and pasted the dry run stuff and it only sort of works..I get a json file, not a video, but more importantly windows throws me a "Working directory lock not supported in windows". Also I have to run it as ./ytdl-sub not just ytdl-sub. ANYWAYS, question was, any plans for a UI? An easy setup for the pleebs? Highly unlikely I know, but doesn't hurt to ask.

**Comment by jmbannon** [comment]
Most likely not, but I hear you, ytdl-sub is not exactly easy to use. I've been thinking about ways to make it easier for folks who just want to simply download a few channels as TV shows by more-or-less just writing a list of URLs. Maybe someday I can build that out

**Reply by Damienlee69** [reply]
That would be sweet. Well, personally I'd like the subscriptions too and If I knew how I'd help, but ya. For now I'll have *sigh* "manually" download with 4K, lol. Good job none the less.

**Comment by jensenkd** [comment]
I could easily see a clone of Sonarr being used as a shell.  It would be a crap-ton of work though.  If only I had more time.

Great project @jmbannon !

**Reply by jmbannon** [reply]
Thanks @jensenkd ! And yes, that would be the dream. Sonarr is so good

**Comment by Damienlee69** [comment]
Ya, I'm quite surprised actually that Sonarr doesn't have the function built or a fork with it by now to be honest. Lidarr has forks with Deezer and Tidal and those are paid services (technically).
